test/com/twu/biblioteca/BibliotecaAppTest.java

```diff
@@ -234,3 +234,4 @@ public void cleanUpStreams() {
         System.setErr(null);
     }
 }
+
```

Comment:
Unit test coverage is good. Do you think these tests exercise the code in a way that an actual user would?